_includes/commands/advtable-cmds.md

```diff
@@ -24,4 +18,22 @@ tinymce.activeEditor.execCommand('mceSortTableByColumnAsc')
 tinymce.activeEditor.execCommand('mceSortTableByColumnDesc')
 tinymce.activeEditor.execCommand('mceSortTableByRowAsc')
 tinymce.activeEditor.execCommand('mceSortTableByRowDesc')
-```
\ No newline at end of file
+tinymce.activeEditor.execCommand('mceTableToggleSeries', false, { name: 'numbers', location: 'left' })
+```
+
+### `mceSortTableAdvanced`
+
+`mceSortTableAdvanced` accepts an object with the following value pairs:
+
+- `sortby` - `'row'` or `'column'`
+- `roworcol` - a zero-indexed integer in a string representing the row from the top of the table or column from the left of the table.
```

Comment:
could this be called startindex or indexstart or just index?
perhaps mention this property is dependent on the 'sortby' property, (thats if we change the property to index)

Reply:
This is out of scope as this is related to `mceSortTableAdvanced` command. If I am looking at it correctly, it would have to be left as 'zero-indexed' as this indicates the first row or column is '0'.